### DISCUSSION TITLE
欢迎来到 Zotero 中文社区！

### DISCUSSION THREAD

**Discussion by northword** [zotero-chinese/wiki#69] [公告 / Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by libin0313** [comment]
微信公众号内容保存到zotero后，附件里面的文字内容总是乱码，该怎么处理呀？同样的方式保存微信公众号内容就会出现乱码，但是保存知乎或者其他网页的内容，就不会出现乱码
<img width="1664" height="1262" alt="微信图片_20250923161333_72" src="https://github.com/user-attachments/assets/98baca53-97fa-4e81-bf93-e845578328a7" />
<img width="2204" height="1024" alt="微信图片_20250923161333_73" src="https://github.com/user-attachments/assets/68599a72-7ce1-4bb8-abee-b390f4cd484c" />
<img width="974" height="380" alt="微信图片_20250923161333_74" src="https://github.com/user-attachments/assets/487b9861-2ab0-4dd1-b644-2f6e65fba008" />
<img width="1016" height="946" alt="微信图片_20250923161333_75" src="https://github.com/user-attachments/assets/5df20c77-df60-4353-a5e9-d9305b90402e" />
<img width="1056" height="1200" alt="微信图片_20250923161333_76" src="https://github.com/user-attachments/assets/50526908-a5bb-4a5d-8d1d-161ff3589f58" />
